docs/Documentation/03a-stoplight-flavored-markdown.md

```diff
@@ -131,104 +139,220 @@ function fibonacci(num){
 
-<!-- title: My Table Title -->
+You can in graphs and diagrams in your markdown via the `mermaid` code block language tag. Here are some examples:
 
-| Tables        |      Are      |  Cool |
-| ------------- | :-----------: | ----: |
-| col 3 is      | right-aligned | $1600 |
-| col 2 is      |    centered   |   $12 |
-| zebra stripes |    are neat   |    $1 |
+<!-- theme: warning -->
 
-### JSON Schema
+> Note that diagrams cannot be nested in other elements like tabs or code groups.
 
-A JSON Schema block is a code fence with either a `json`, `yaml`, or `yml` language tag, plus an additional `json_schema` language tag. The contents of the code fence should be the JSON Schema object to be rendered. 
+### Flowchart
 
-For example the Markdown:
+```mermaid
+graph TB
+    c1-->a2
+    subgraph ide1 [one]
+    a1-->a2
+    end
+```
 
-````md
-```json json_schema
-{
-    "title": "User",
-    "type": "object",
-    "properties": {
-        "id": {
-            "type": "string"
-        },
-        "name": {
-            "type": "string",
-            "description": "The user's full name."
-        },
-        "age": {
-            "type": "number",
-            "minimum": 0,
-            "maximum": 150
-        }
-    },
-    "required": [
-        "id",
-        "name"
-    ]
-}
-\```
-````
+```example
+graph TB
+    c1-->a2
+    subgraph ide1 [one]
+    a1-->a2
+    end
+```
 
-<!-- theme: warning -->
 
-> In the example above, be sure to remove the `\` that precedes the last three backticks at the end of the code fence. This was included for demonstration purposes only.
+### Sequence
+
+```mermaid
+sequenceDiagram
+    autonumber
+    Alice->>John: Hello John, how are you?
+    loop Healthcheck
+        John->>John: Fight against hypochondria
+    end
+    Note right of John: Rational thoughts!
+    John-->>Alice: Great!
+    John->>Bob: How about you?
+    Bob-->>John: Jolly good!
+```
+
+```example
+sequenceDiagram
+    autonumber
+    Alice->>John: Hello John, how are you?
+    loop Healthcheck
+        John->>John: Fight against hypochondria
+    end
+    Note right of John: Rational thoughts!
+    John-->>Alice: Great!
+    John->>Bob: How about you?
+    Bob-->>John: Jolly good!
+```
+
+### Journey
+
+```mermaid
+journey
+    title My working day
+    section Go to work
+      Make tea: 5: Me
+      Go upstairs: 3: Me
+      Do work: 1: Me, Cat
+    section Go home
+      Go downstairs: 5: Me
+      Sit down: 5: Me
+```
+
+```example
+journey
+    title My working day
+    section Go to work
+      Make tea: 5: Me
+      Go upstairs: 3: Me
+      Do work: 1: Me, Cat
+    section Go home
+      Go downstairs: 5: Me
+      Sit down: 5: Me
+```
+
+
+## JSON Schema
 
-The example above results in:
+The SMD JSON schema block is an MD code block with an additional `json_schema` language tag. The contents of the code
+fence should be the JSON schema object to be rendered. The primary language tag can be `YAML, `YML`, or `JSON.
```

Comment:
```suggestion
fence should be the JSON schema object to be rendered. The primary language tag can be `YAML, `YML`, or `JSON`.
```

Reply:
Did you mean to resolve this? It looks strange if you don't add the extra ` after JSON